src/lib.rs

```diff
@@ -1107,6 +1107,49 @@ pub unsafe trait TryFromBytes {
         candidate: Maybe<'_, Self, A>,
     ) -> bool;
 
+    /// Is a given source a valid instance of `Self`?
+    ///
+    /// # Safety
+    ///
+    /// Unsafe code may assume that, if `is_mut_src_valid(src)` returns true,
+    /// `*src` is a bit-valid instance of `Self`, and that the size of `Src` is
+    /// greater than or equal to the size of `Self`.
+    ///
+    /// # Panics
+    ///
+    /// `is_src_valid` panics under exactly the same circumstances as
+    /// [`is_bit_valid`].
```

Comment:
Maybe also mention that size mismatch can cause a panic or a compilation failure? (just like we do for `is_bit_valid`)

Reply:
[Done](https://github.com/google/zerocopy/compare/80bdd61e29970f8ba17c2ce5e47f2c69b1d9dffa..dca55262906a36d52de53f88a608eb34d56c1c27#diff-bbffbe0cf99afd2b815401c126c4790999e882fb64e3cb367aea899c64c95966R425-R429)